### ISSUE TITLE
support rfc6749?

### ISSUE THREAD

**Issue by Guoxweii** [nxtbgthng/OAuth2Client#147]
the rfc url: http://tools.ietf.org/html/rfc6749

The difference between old OAuth2.0 spec and new one: http://developer.mixi.co.jp/en/connect/mixi_graph_api/diff_of_oauth_spec/

it will upgrade to supported rfc6749?

**Comment by toto** [comment]
Yes, this should be done. Most of the features of RFC6749 should be working with a little bit of configuration, but indeed the RFC should be our next goal.

**Comment by Guoxweii** [comment]
Thanks for your reply.  
- file:  NXOAuth2Connection.m 
- line: 421
  
  ```
    [authenticateHeader rangeOfString:@"expired_token"].location != NSNotFound
  ```

but the rfc6749 can't support this head, now the heas is: 

```
   WWW-Authenticate: Bearer  realm="api.mixi-platform.com", error="invalid_token", error_description="The access token expired"
```

so can i  configuration this?
